[jit] small cleanups after script:: removal

suo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#34580 [jit] small cleanups after script:: removal**

1. Remove remaining uses of `script::` namespace from the codebase,
including the typedefs we left in for BC.
2. Add a deprecation warning to the typedefs.
3. Add one more typedef for `script::ExtraFilesMap` which is part of the
public interface.

Differential Revision: [D20403143](https://our.internmc.facebook.com/intern/diff/D20403143)

suo committed:
[jit] small cleanups after script:: removal

1. Remove remaining uses of `script::` namespace from the codebase,
including the typedefs we left in for BC.
2. Add a deprecation warning to the typedefs.
3. Add one more typedef for `script::ExtraFilesMap` which is part of the
public interface.

[ghstack-poisoned]

eellison reviewed (approved):
LGTM. Hoping that this doesn't add a bunch of deprecation warnings to the build...

suo commented:
This PR also removes all remaining uses internally, so there should be no warnings in the build.

suo committed:
Update on "[jit] small cleanups after script:: removal"


1. Remove remaining uses of `script::` namespace from the codebase,
including the typedefs we left in for BC.
2. Add a deprecation warning to the typedefs.
3. Add one more typedef for `script::ExtraFilesMap` which is part of the
public interface.

[ghstack-poisoned]

suo committed:
Update on "[jit] small cleanups after script:: removal"


1. Remove remaining uses of `script::` namespace from the codebase,
including the typedefs we left in for BC.
2. Add a deprecation warning to the typedefs.
3. Add one more typedef for `script::ExtraFilesMap` which is part of the
public interface.

Differential Revision: [D20403143](https://our.internmc.facebook.com/intern/diff/D20403143)

[ghstack-poisoned]

suo commented:
closing in favor of https://github.com/pytorch/pytorch/pull/34677
